add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/DoubleFlutterActivity.kt

```diff
@@ -0,0 +1,76 @@
+package dev.flutter.multipleflutters
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import androidx.fragment.app.FragmentActivity
+import androidx.fragment.app.FragmentManager
+import io.flutter.embedding.android.FlutterFragment
+import io.flutter.embedding.engine.FlutterEngineCache
+
+/**
+ * An activity that displays 2 FlutterFragments vertically.
+ */
+class DoubleFlutterActivity : FragmentActivity(), EngineBindingsDelegate {
+    private val topBindings : EngineBindings by lazy {
+        EngineBindings(this, this, "topMain")
+    }
+    private val bottomBindings : EngineBindings by lazy {
```

Comment:
I think the lateinit val pattern might be a bit clearer (since it's less ambiguous by what time they become not null). Lazy is better for things we might indeterministically (or never) access, which isn't the case here.

Reply:
There is no such thing as `latinit val`, there is only `lateinit var`, that's why I think lazy is better here.  Kotlin doesn't have the idea of `final` either.  From a user's perspective they should never need to think about the value ever being null.